server/push_notification.go

```diff
@@ -51,6 +51,7 @@ type PushNotification struct {
 	OverrideIconURL  string `json:"override_icon_url"`
 	FromWebhook      string `json:"from_webhook"`
 	Version          string `json:"version"`
+	IsCRTEnabled     bool   `json:"is_crt_enabled"`
```

Comment:
Previous servers won't include a value for this in the payload that the proxy receives, right? In that case should we omit here and then ser the default value to false?

Reply:
Value sent from server is `false` by default for every push notification 🙂

Reply:
I doubt a 5.35 server sends that prop as falsey

Reply:
Sorry I was telling you about  the server PR. `is_crt_enabled` was never sent at all previously.

Wouldn't this automatically become `false` with this? https://github.com/mattermost/mattermost-push-proxy/blob/1016c30ba8a5476ab78582f0bb64196bd5da1781/server/push_notification.go#L65

Reply:
That is exactly why I asked, does golang set the default value as false? As idk if that is in fact the case, if it is, then we should be good

Reply:
Yes had the  same doubt and tested in playground to confirm it https://play.golang.org/p/MJILfwMwWWi 😅

Reply:
Yes, the default value of a bool is `false`.